core/controllers/reader.py

```diff
@@ -742,6 +742,32 @@ class LearnerIncompleteActivityHandler(base.BaseHandler):
     the user.
     """
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'activity_type': {
+            'schema': {
+                'type': 'basestring',
+                'choices': [
+                    constants.ACTIVITY_TYPE_EXPLORATION,
```

Comment:
I have a question about this. Can you point to parts of the code where this handler is actually pinged with the other choices of constants you have below? I want to be sure that all constants here are actually used with this handler somewhere in the codebase, before approving.

Reply:
Those choices are pinged [here](https://github.com/oppia/oppia/blob/b3f57960a409df0cd0d3e1c60be47d105f638ca3/core/controllers/reader.py#L781-L792). Please have a look and let me know if you want me to do anything else. Thanks :)